custom_components/cz_energy_spot_prices/spot_rate.py

```diff
@@ -131,6 +132,8 @@ async def get_gas_rates(self, start: datetime, in_eur: bool, unit: EnergyUnit) -
 
     async def _get_rates(self, query: str, unit: Literal['kWh', 'MWh'], has_hours: bool = True) -> RateByDatetime:
         text = await self._download(query)
+        if 'Application is not available' in text:
```

Comment:
I'm not sure about this approach. This message is implementation specific and might not be there during next outage. Do you know it OTE was returning HTTP status code >= 400 during the outage? It would be better to detect that than relying on text it returns.

Reply:
jak už jsem psal v Issue:

> Nevracelo žadnej HTTP kód, jen na webu psali viz můj PR. To byl jediný způsob jak to detekovat. Ddpověď byla normálně 200 s popisem v obsahu. Proto tvoje integrace házela chybu že neumí parsovat tu odpověď.

Proto jsem ten check napsal tak jak jsem ho napsal. Mělo to bohužel důvod. 😉

Reply:
maybe, it'd be better to try to parse the response.. and if parsing fails => raise the `UpdateFailed`

Reply:
Yeah but with that people will still wonder why. :) Or at least I would. With that check you at least know why.